src/modules/Builder/features/ActivityItems/ItemConfiguration/ItemConfiguration.utils.tsx

```diff
@@ -19,7 +19,7 @@ import {
   DEFAULT_AUDIO_DURATION_SEC,
   SELECTION_OPTIONS_COLOR_PALETTE,
 } from './ItemConfiguration.const';
-import { getEmptyCondition } from '../../ActivityItemsFlow/ItemFlow/ItemFlow.utils';
+import { getEmptyCondition } from '../../ActivityItemsFlow_old/ItemFlow/ItemFlow.utils';
```

Comment:
As I see here the function is only importing from the old utils.
What if we turn the flag 'on'. Maybe it will break the code flow?

Reply:
I will revert it to `ActivityItemsFlow`. There are no differences in logic in `ItemFlow.utils';` in `ActivityItemsFlow` and `ActivityItemsFlow_old` folders. So after approving and testing on next release the `ActivityItemsFlow_old`  will be removed.